### DISCUSSION TITLE
Add TorusShape3D for collisions

### DISCUSSION THREAD

**Discussion by venilark** [godotengine/godot-proposals#6522] [3D]
There is TorusMesh, CSGTorus but no TorusShape3D, how is it?
I tried to implement a ring that gets bigger over time while keeping the same thickness but that meant changing the collision shape and I couldn't do it,  or didn't know how, so I had to rely on the CSGTorus and check the collision with it with another Area3D, a really bad solution that wouldn't be necessary with this new collision shape.

**Comment by AThousandShips** [comment]
Duplicate of #6244
